src/_locales/ja/messages.json

```diff
@@ -0,0 +1,723 @@
+    },
+    "intro_link_policy": {
+        "message": "プライバシーポリシー",
+        "description": "Shown at the bottom of the intro page, links to the EFF software and technology projects privacy policy."
+    },
+    "report_input_label": {
+        "message": "説明",
+        "description": ""
+    },
+    "not_yet_blocked_header": {
+        "message": "Privacy Badgerは、これらのドメインがブロックされるべきかどうか、まだ決定していません。",
+        "description": "Popup domain list header text; separates blocked from haven't-yet-seen-enough-to-block potential trackers."
+    },
+    "options_show_not_yet_blocked": {
+        "message": "Privacy Badgerがまだブロックすると決めていないドメインを表示する:",
+        "description": "Label for a checkbox on the Tracking Domains options page tab. Should match wording used in the 'not_yet_blocked_header' message."
+    },
+    "non_tracker": {
+        "message": "以下のドメインは、あなたを追跡していないように見えます。",
+        "description": "Header text; separates tracking from non-tracking domains in the popup."
+    },
+    "popup_options_button": {
+        "message": "オプション",
+        "description": ""
+    },
+    "share_button_title_facebook": {
+        "message": "Facebookで共有",
+        "description": "Text that comes up when you hover over the social sharing buttons on the intro page."
+    },
+    "report_fail": {
+        "message": "すみません。何らかの問題が発生しました。",
+        "description": ""
+    },
+    "social_tooltip_pb_has_replaced": {
+        "message": "Privacy Badger has replaced this $BUTTON$ button",
+        "description": "Tooltip shown over a replaced social button. For example, \"Privacy Badger has replaced this Facebook Like button\". See also the widget_placeholder_pb_has_replaced message.",
+        "placeholders": {
+            "button": {
+                "content": "$1",
+                "example": "Facebook Like"
+            }
+        }
+    },
+    "widget_placeholder_pb_has_replaced": {
+        "message": "Privacy Badgerは、 $START_ANCHOR_TAG$この $WIDGET$ ウィジェット$END_ANCHOR_TAG$ を置き換えました。",
+        "description": "Text shown inside a replaced widget's placeholder. For example, \"Privacy Badger has replaced this Google reCAPTCHA widget\". See also the social_tooltip_pb_has_replaced message.",
+        "placeholders": {
+            "widget": {
+                "content": "$1",
+                "example": "Google reCAPTCHA"
+            },
+            "start_anchor_tag": {
+                "content": "$2",
+                "example": "<a href='https://vimeo.com/76979871'>"
+            },
+            "end_anchor_tag": {
+                "content": "$3",
+                "example": "</a>"
+            }
+        }
+    },
+    "allow_once": {
+        "message": "今回だけ許可",
+        "description": "Button in the placeholder shown in place of certain third-party (video, audio, commenting) widgets. Clicking it will allow the third-party widget to load this one time only."
+    },
+    "allow_on_site": {
+        "message": "このサイトでは常に許可",
```

Comment:
This and the previous message (`allow_once`) are buttons the user can click to activate a replaced widget. Just checking, is the wording appropriate for buttons?

Reply:
This is a bit confusing wording and I would amend it to the proper wording